### DISCUSSION TITLE
Embed one drawing in another

### DISCUSSION THREAD

**Discussion by dcoales** [zsviczian/obsidian-excalidraw-plugin#1195] [Q&A]
I'e seen people talking about embedding one excalidraw drawing inside another but can't figure out how to do this.  I suspect I'm missing something obvious but any pointers would be gratefully received. Thanks.

**Comment by dcoales** [comment]
I think I see the problem.  The diagram is actually adding as an SVG.  What would be great is if double clicking the SVG opened the original drawing in another tab.

**Comment by eresdf** [comment]
Hello! Did you find a solution?

**Comment by Sandr-aa** [comment]
You can find the option in the sidebar, either directly or in the obsidian tools menu. After selecting your file via the file path, it is pasted as an linked image (updating the embedded file will also update it's image representation)
![2024-08-08_19-08](https://github.com/user-attachments/assets/46bbcd70-88a8-41ed-8796-46f8195d59d3)

**Reply by himra13** [reply]
thank you, it works really well
